squad/core/comparison.py

```diff
@@ -108,37 +108,66 @@ def diff(self):
         return self.__diff__
 
     __regressions__ = None
+    __progressions__ = None
```

Comment:
I don't recall ever hearing this term progression in this context. is it common?

maybe we want to call this "fixes"?

Reply:
We came up with this with @danrue. I'm OK calling it 'fixes' or whatever else that can be easily recognized.